### DISCUSSION TITLE
播放页样式可以像acfun那样吗？

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#5012] [功能建议 / Ideas]
B站默认播放器窗口很小，网页两边浪费了很多空间。宽屏模式还会遮挡视频，必须手动拉滚动条。但是a站宽屏是和youtube类似的。没有这个痛点。
![图片](https://github.com/user-attachments/assets/bc954376-c014-4008-9723-0af99c594900)
![图片](https://github.com/user-attachments/assets/8008c84b-1802-42a8-9351-1fef8e5b91c6)
![图片](https://github.com/user-attachments/assets/fe46ce18-ff0a-4bc0-9231-98b12f5d0421)

**Comment by Geeyun-JY3** [comment]
重复：#4697
推荐该讨论下提到的这个脚本：https://github.com/the1812/Bilibili-Evolved/discussions/4697#discussioncomment-8923305

**Comment by festoney8** [comment]
可以调，差不多普通模式能调到和宽屏一样大，这个功能在播放页插件菜单的"播放设定"里
https://greasyfork.org/zh-CN/scripts/479861

**Reply by ghost** [reply]
> 播放页插件菜单的"播放设定"里

可以的，舒服了很多，就差个剧场模式了
![图片](https://github.com/user-attachments/assets/7aea7f45-f624-461c-aa8c-a7ada877dada)
